docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md

```diff
@@ -183,7 +183,7 @@ COPY build.gradle /home/gradle
 
 RUN gradle copyDependencies
 
-FROM keycloak/keycloak:21.1 as builder
+FROM keycloak/keycloak:22.0 as builder
```

Comment:
Renovate ftw m)

Reply:
doubt it's gonna happen in this repo

Reply:
also current version is 23.x and that one is not compatible with Camunda ... so renovate ftw maybe not.